databases/week3/meal_share.sql

```diff
@@ -0,0 +1,74 @@
+CREATE DATABASE meal_share_db
+ DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE meal_share_db;
+CREATE TABLE `meal` (
+ `id` INT(10) unsigned NOT NULL AUTO_INCREMENT,
+ `title` VARCHAR(50) NOT NULL,
+ `description` TEXT NULL,
+ `location` VARCHAR(50) NOT NULL,
+ `when` DATETIME NOT NULL, 
+ `max_reservation` INT(10) unsigned NOT NULL,
+ `price` DECIMAL(10,2) NOT NULL,
+ `created` DATE NOT NULL, 
+ PRIMARY KEY(`id`)
+)
+
+DESCRIBE  meal;
+
+CREATE TABLE `reservations` (
```

Comment:
On the table names stick to either plural or singular, but please don't mix up. e.g. either use `meal`  and `reservation`  or `meals` and `reservations`. Otherwise it gets confusing :/